aerleon/lib/openconfig.py

```diff
@@ -68,6 +68,15 @@ def RecursiveDict() -> DefaultDict[Any, Any]:
     "ACLEntry",
     {"sequence-id": int, "actions": Action, "ipv4": IP, "ipv6": IP, "transport": Transport},
 )
+ACLSet = TypedDict(
+    "ACLSet",
+    {
+        "acl-entries": {"acl-entry": list[ACLEntry]},
```

Comment:
I think your issue is here.
You are writing 
```"acl-entries": {"acl-entry": list[ACLEntry]},```

This is saying the key is a value of another dict. What TypeDict wants is only keys equalling the type of the value such as above you can see ACLEntry has 

```
"actions": Action,
```
and above `Action` is defined as another TypedDict.

You will likely need to do this for both `acl-entries` and `config` as I believe both are wrong.

Reply:
Fixed, thanks!